services/monarch-link-app/requirements.txt

```diff
@@ -0,0 +1,15 @@
+click==8.1.3
+colorama==0.4.6
+Flask==2.2.2
+Flask-Cors==4.0.2
+Flask-Login==0.6.2
+Flask-SQLAlchemy==3.0.2
+greenlet==2.0.1
+itsdangerous==2.1.2
+Jinja2==3.1.2
+MarkupSafe==2.1.1
+six==1.16.0
+SQLAlchemy==1.4.43
+SQLAlchemy-Utils==0.38.3
+Werkzeug==2.2.2
+gunicorn==22.0.0
```

Comment:
Suggest setting only what you use directly with semver and allowing some newer verisons. `pip freeze` output is almost always overly restrictive.